css-nav-1/Overview.bs

```diff
@@ -1089,17 +1089,18 @@ run the following steps:
     then return it.
 3. Otherwise, return <var>searchOrigin</var>.
 
-If the status of <var>old focus target</var> changed not by the spatial navigation,
+If the status of <var>old focus target</var> changed not by moving the focus,
 <dfn lt="update the search origin | updating the search origin">update the <a>search origin</a></dfn> as following:
-1. If <var>old focus target</var> becomes
+1. If <var>old focus target</var> is
```

Comment:
"old focus target" may be confusing.
But when it is removed or disabled, the active element will be the Document's viewport.
So the sentence like "if activeElement is removed" seems unnatural to me.
That's the reason why I put "old" in the naming of the argument of this step.

How about "previous activeElement" or "previous focus target"?
(HTML Spec uses "focus target" instead of "active element" when it refers to the element which has the focus.)

Reply:
Hmm. Ok. Yeah, I don't know. As a JS and C++ developer, when I read _activeElement_, I directly get what we're talking about. But yeah, "focus target" is perhaps more correct in a spec... Not sure. In general I think the spec is a bit too wordy :P In speech we'd just say "focus"? :)

Reply:
I think "focus target" is more precise than just "focus". Because in this description, the object which becomes disappeared or disabled is the element, not the focus itself.  ('focus' will remain somewhere in the document)

Yeah, the spec is a bit wordy, but I think precise description is also important.